.travis.yml

```diff
@@ -6,23 +6,35 @@ branches:
   only:
   - develop
   - prod
-  - strides
 install:
-  - wget https://github.com/Sage-Bionetworks/infra-utils/archive/master.zip -O /tmp/infra-utils.zip
-  - unzip -j -n /tmp/infra-utils.zip -x "infra-utils-master/.gitignore" "infra-utils-master/LICENSE" "infra-utils-master/*.md" "infra-utils-master/aws/*"
-  - ./setup_aws_cli.sh || travis_terminate 1
-  - pip install pre-commit sceptre sceptre-ssm-resolver travis-wait-improved
+  - pip install awscli pre-commit sceptre sceptre-ssm-resolver travis-wait-improved
```

Comment:
Why do you need to use travis-wait-improved? Just curious.

Reply:
we use it because beanstalk deployments can take a long time travis-wait-improved helps us debug the deployment in case it fails.